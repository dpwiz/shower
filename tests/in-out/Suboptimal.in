[ ( "Units are widened", () ),
  ( "Chars are renormalized", '\x01', '\x02' ),
  ( "String gaps are broken", "with \  \ gaps" ),
  ( "Unboxed tuples have spaces added", (# 1, 2 #) ),
  ( "Unboxed strings have spaces added", "hello"# ) ]
