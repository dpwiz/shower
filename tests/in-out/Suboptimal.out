[ ( "Units are widened", ( ) ),
  ( "Chars are renormalized", '\SOH', '\STX' ),
  ( "String gaps are broken", "with \\  \\ gaps" ),
  ( "Unboxed tuples have spaces added", ( # 1, 2 # ) ),
  ( "Unboxed strings have spaces added", "hello" # ) ]
